code/datums/diseases/advance/symptoms/disfiguration.dm

```diff
@@ -1,50 +0,0 @@
-/*
```

Comment:
why remove this symptom?

Reply:
mainly because it was added as a replacement to vitiligo/revitiligo, so I just removed it with the rest when I added the two back in

Reply:
this is a more interesting symptom IMO, so I'd like to keep it as well. though I think I will have to ask some viro players for opinions about this.

Reply:
yeah ok I added it back